# HG changeset patch
# User Rik <rik@octave.org>
# Date 1361985986 28800
#      Wed Feb 27 09:26:26 2013 -0800
# Node ID a885686b732a4402d5c72bf0036e1f7632e81b33
# Parent  ed36d5543b27d097f7dab5cf314699b0d942515d
build: Shorten long doc node title to prevent error in creating tar archive.

* doc/interpreter/octave.texi, doc/interpreter/plot.txi: Shorten node name
by removing @code{} macros from araound axis, line, patch words.

diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -493,17 +493,17 @@ Plotting
 
 High-Level Plotting
 
 * Two-Dimensional Plots::       
 * Three-Dimensional Plots::  
 * Plot Annotations::            
 * Multiple Plots on One Page::  
 * Multiple Plot Windows::       
-* Use of @code{axis}@comma{} @code{line}@comma{} and @code{patch} Functions
+* Use of axis@comma{} line@comma{} and patch Functions::
 * Manipulation of Plot Windows::
 * Use of the @code{interpreter} Property::
 * Printing and Saving Plots::              
 * Interacting with Plots::      
 * Test Plotting Functions::     
 
 Two-Dimensional Plots
 
diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -57,17 +57,17 @@ If you need more detailed control, see @
 and @ref{Advanced Plotting}.
 
 @menu
 * Two-Dimensional Plots::       
 * Three-Dimensional Plots::  
 * Plot Annotations::            
 * Multiple Plots on One Page::  
 * Multiple Plot Windows::       
-* Use of @code{axis}@comma{} @code{line}@comma{} and @code{patch} Functions::
+* Use of axis@comma{} line@comma{} and patch Functions::
 * Manipulation of Plot Windows::
 * Use of the @code{interpreter} Property::
 * Printing and Saving Plots::              
 * Interacting with Plots::      
 * Test Plotting Functions::     
 @end menu
 
 @node Two-Dimensional Plots
@@ -516,18 +516,18 @@ fplot (@@cos, [-10, 10]);
 @end example
 
 @noindent
 creates two figures, with the first displaying a sine wave and
 the second a cosine wave.  Figure numbers must be positive integers.
 
 @DOCSTRING(figure)
 
-@node Use of @code{axis}@comma{} @code{line}@comma{} and @code{patch} Functions
-@subsection Use of @code{axis}@comma{} @code{line}@comma{} and @code{patch} Functions
+@node Use of axis@comma{} line@comma{} and patch Functions
+@subsection Use of axis@comma{} line@comma{} and patch Functions
 
 You can create axes, line, and patch objects directly using the
 @code{axes}, @code{line}, and @code{patch} functions.  These objects
 become children of the current axes object.
 
 @DOCSTRING(axes)
 @DOCSTRING(line)
 @DOCSTRING(patch)
